ogr/parsing.py

```diff
@@ -136,16 +136,21 @@ def parse_git_repo(potential_url: str) -> Optional[RepoUrl]:
         )
     if not username and len(split) >= 2:
         # path contains username/reponame
+        # or some/namespace/reponame
+        # or fork/username/some/namespace/reponame
 
-        is_fork = "fork" in split
-        if is_fork and len(split) == 4:
+        is_pagure_fork = "fork" in split
+        if is_pagure_fork and len(split) == 4:
```

Comment:
Why '4'? What happens with https://pagure.io/fork/csomh/waiverdb ?

Reply:
Yes, good point. Some historical relict from the Betka days... Fixed.

Reply:
How about adding atest for `https://host.name/fork/username/repo`?